timers.js

```diff
@@ -1,4 +1,54 @@
+var timer = {
+  start:  document.getElementById('start'),
+  reset:  document.getElementById('reset'),
+  stop:   document.getElementById('start'),
+  pause:  document.getElementById('pause'),
+  timerElement:  document.getElementById('timer'),
+  seconds: 0,
+  timerID: 0,
 
+  changeName: function(){
+    this.start.textContent = "Stop";
+  },
+  changeBack: function (){
+    this.start.textContent = "Start";
+  },
+  clickStart: function (){
+    this.start.addEventListener("click", function(){
+          if (timer.start.textContent === "Start"){
+            timer.timerID = setInterval(function(){
```

Comment:
It might be clearer if we move this anonymous function into a named function like `incrementSeconds`.